### PR TITLE
Feature get packages

### PR DESCRIPTION
**Pull request by a-cool-train**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cmd/eksctl-anywhere/cmd/getpackage.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+func init() {
+	getCmd.AddCommand(getPackageCommand)
+	getPackageCommand.Flags().StringVarP(&gpo.output, "output", "o", "", "Specifies the output format (valid option: json, yaml)")
+}
+
+var getPackageCommand = &cobra.Command{
+	Use:     "package(s)",
+	Aliases: []string{"package", "packages"},
+	Short:   "Get package(s)",
+	Long:    "This command is used to display one  or more curated packages",
```
Comment (TerryHowe) [inline cmd/eksctl-anywhere/cmd/getpackage.go:29]:
```suggestion
	Long:    "This command is used to display the installed packages",
```

File: pkg/executables/kubectl.go [hunk 2]
```diff
@@ -1222,3 +1222,27 @@ func (k *Kubectl) GetDaemonSet(ctx context.Context, name, namespace, kubeconfig
 
 	return obj, nil
 }
+
+func (k *Kubectl) GetPackagesFromKubectl(ctx context.Context, packageInstanceName, kubeconfig string, output string) (string, error) {
+	params := []string{
+		"get", "packages", "--kubeconfig", kubeconfig, "--namespace", constants.EksaPackagesName,
```
Comment (TerryHowe) [inline pkg/executables/kubectl.go:1228]:
```suggestion
		"get", "packages",  "--namespace", constants.EksaPackagesName,
```

Comment (a-cool-train) [reply pkg/executables/kubectl.go:1228]: I tested this and it looks like kubeconfig is required for the command to work.

Comment (TerryHowe) [reply pkg/executables/kubectl.go:1228]: Not if you set the KUBECONFIF env var or use the default location

**Comment by TerryHowe** [comment]
Close this PR

**Pull request closed without merging**